asset-transfer-basic/rest-api-typescript/package.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Can I recommened adding `source-map-support` (cf https://github.com/hyperledgendary/weftility/blob/6fea022ad42dadd0dd16b060448b596481295827/package.json#L43)

With a setup line in the main ts file it means all stack traces refer to the typescript not javascript. (MGK pointed me at this one)

Reply:
```
import sourceMapSupport from 'source-map-support';
sourceMapSupport.install();
```

Reply:
Top tip, thanks